Assets/PatchKit Patcher/Scripts/UI/Languages/PatcherLanguages.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using Newtonsoft.Json;
+using PatchKit.Unity.Patcher.Debug;
+using PatchKit.Unity.Utilities;
+using UnityEngine;
+
+namespace PatchKit.Unity.UI.Languages
+{
+    public class PatcherLanguages : MonoBehaviour
+    {
+        public static PatcherLanguages Instance;
```

Comment:
If every method & property in the class would be `static` then this becomes unnecessary.

Reply:
done